pwndbg/commands/context.py

```diff
@@ -1,3 +1,6 @@
+#ifndef CONTEXT_PY
+#define CONTEXT_PY
```

Comment:
Seems like IDE/automatic snippet inclusion occurred? :)

It also happens at the end :p

Reply:
Yep. Somehow vim thought it to be c. Sorry for that.